### PR TITLE
dict: add page

### PR DESCRIPTION
**Pull request by qvieth**
create dictd cheat sheet

- [x] The page (if new), does not already exist in the repository.
- [x] The page is in the correct platform directory (`common/`, `linux/`, etc.)
- [x] The page has 8 or fewer examples.
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).
- [x] The page follows the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The page description includes a link to documentation or a homepage (if applicable).

### CODE REVIEW

**Comment by qvieth** [comment]
https://www.putorius.net/linux-command-line-dictionary.html#installation-on-red-hat-based-systems

https://wiki.archlinux.org/title/dictd

**Comment by qvieth** [comment]
a useful cli for query words definition and thesarus

**Comment by CleanMachine1** [comment]
Do you mind fixing the page, there is a lot of syntax errors.

Take a look at this, an example page

```
# git cp

> Copy an existing file to a new location, preserving history.
> Part of `git-extras`.
> More information: <https://github.com/tj/git-extras/blob/master/Commands.md#git-cp>.

- Copy an existing file in a Git repo, staying in the same directory:

`git cp {{file}} {{new_file}}`

- Copy an existing file in a Git repo and place it elsewhere:

`git cp {{path/to/file}} {{path/to/new_file}}`
```

**Comment by qvieth** [comment]
Oh so sorry, I thought we use the same format as cht.sh guys.

**Review by qvieth — COMMENTED** [review]
fix wrong syntax

**Review by qvieth — COMMENTED** [review]
fix syntax error

**Comment by qvieth** [comment]
thank you, @navarroaxel

File: pages/common/dict.md [hunk 1]
```diff
@@ -0,0 +1,24 @@
+# dict
+
+> Client/server software, human language dictionary databases, and tools supporting the DICT protocol.
```
Comment (marchersimon) [inline pages/common/dict.md:3]: I think that description is very vague and dosn't really tell what the command is about. I know that's what the official page uses, but could we maybe make this a little more concrete?

File: pages/common/dict.md [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+> More information: <https://github.com/cheusov/dictd>.
+
+- List available databases:
+
+`dict -D`
```
Comment (marchersimon) [inline pages/common/dict.md:8]: Does that command have long options?

File: pages/common/dict.md [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+
+- Get database information:
```
Comment (marchersimon) [inline pages/common/dict.md:10]: We generally try to use more verbose descriptions.
```suggestion
- Get information about a database:
```

File: pages/common/dict.md [hunk 4]
```diff
@@ -0,0 +1,24 @@
...
+
+`dict -i {{database_name}}`
+
+- Look up word in database:
```
Comment (marchersimon) [inline pages/common/dict.md:14]:
```suggestion
- Look up a word in a specific database:
```

File: pages/common/dict.md [hunk 5]
```diff
@@ -0,0 +1,24 @@
...
+
+`dict -d {{database_name}} {{word}}`
+
+- Look up word in all available databases:
```
Comment (marchersimon) [inline pages/common/dict.md:18]:
```suggestion
- Look up a word in all available databases:
```

File: pages/common/dict.md [hunk 6]
```diff
@@ -0,0 +1,24 @@
...
+> Client/server software, human language dictionary databases, and tools supporting the DICT protocol.
+> More information: <https://github.com/cheusov/dictd>.
+
+- List available databases:
+
+`dict -D`
+
+- Get database information:
+
+`dict -i {{database_name}}`
```
Comment (marchersimon) [inline pages/common/dict.md:12]: Do you just need the name or also the path?

File: pages/common/dict.md [hunk 7]
```diff
@@ -0,0 +1,24 @@
...
+
+- Look up word in database:
+
+`dict -d {{database_name}} {{word}}`
+
+- Look up word in all available databases:
+
+`dict {{word}}`
+
+- Show server info:
```
Comment (marchersimon) [inline pages/common/dict.md:22]:
```suggestion
- Show information about the DICT server:
```
Not sure if `DICT` is correct, but it should be something like that

File: pages/common/dict.md (hunk 1 above)
Comment (qvieth) [reply pages/common/dict.md:3]: I looked up dictd but not really grasp all of its feature, I was afraid making short definition wouldn't fully cover or make the wrong assumption about the dictd package itself. But for the dictionary feature, it's really useful.

File: pages/common/dict.md (hunk 2 above)
Comment (qvieth) [reply pages/common/dict.md:8]: No, it doesn't

File: pages/common/dict.md (hunk 4 above)
Comment (qvieth) [reply pages/common/dict.md:14]: That's why I need a dictionary command XD, thank you, @marchersimon.

File: pages/common/dict.md (hunk 6 above)
Comment (qvieth) [reply pages/common/dict.md:12]: Just the name, which comes from using `dict -D`

File: pages/common/dict.md (hunk 7 above)
Comment (qvieth) [reply pages/common/dict.md:22]: Yes!! Thank you, @marchersimon

File: pages/common/dict.md [hunk 8]
```diff
@@ -0,0 +1,24 @@
+# dict
+
+> Command line dictionary, using DICT protocol.
```
Comment (marchersimon) [inline pages/common/dict.md:3]:
```suggestion
> Command line dictionary using the DICT protocol.
```

**Review by marchersimon — COMMENTED** [review]
Just one last thing

File: pages/common/dict.md (hunk 8 above)
Comment (qvieth) [reply pages/common/dict.md]: Yay, finally XD, thank you so much @marchersimon.

**Review by navarroaxel — APPROVED** [review]

**Review by CleanMachine1 — APPROVED** [review]
LGTM

**Pull request merged**